projects/ngrx.io/content/guide/store/reducers.md

```diff
@@ -156,10 +156,10 @@ export const scoreboardFeatureKey = 'game';
 import { NgModule } from '@angular/core';
 import { StoreModule } from '@ngrx/store';
 import * as fromScoreboard from './reducers/scoreboard.reducer';
-
+  
 @NgModule({
   imports: [
-    StoreModule.forFeature(fromScoreboard.scoreboardFeatureKey, fromScoreboard.reducer)
+    StoreModule.forFeature(fromScoreboard.scoreboardFeatureKey, fromScoreboard.scoreboardReducer)
```

Comment:
I would keep the same style as the previous snippet.

```suggestion
    StoreModule.forFeature(fromScoreboard.scoreboardFeatureKey, scoreboardReducer)
```

Reply:
You're right, but then we need to import the scoreboardFeatureKey as well.